app/serializers/insecure_user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class InsecureUserSerializer < ActiveModel::Serializer
```

Comment:
This was also part of the serializer guide. I wasn't sure if I needed it until after I created it. I don't actually need it, but wasn't sure if I could delete it without affecting the database migrations. I think I should be ok to delete it. 

> On Dec 29, 2015, at 9:09 PM, David Roberts <notifications@github.com> wrote:
> 
> In app/serializers/insecure_user_serializer.rb:
> 
> > @@ -0,0 +1,7 @@
> > +class InsecureUserSerializer < ActiveModel::Serializer
> What's this about?
> 
> —
> Reply to this email directly or view it on GitHub.
>